src/lang/ar/exceptions.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
This file need translation.

Reply:
the final user of the package is the one who is responsible for that